src/main/java/db/migration/CommonDbUtils.java

```diff
@@ -0,0 +1,23 @@
+package db.migration;
+
+import org.flywaydb.core.api.migration.Context;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class CommonDbUtils {
+
+    public static void insertFile(Context context, String name, String file) throws IOException {
+        JdbcTemplate jdbcTemplate = new JdbcTemplate(context.getConfiguration().getDataSource());
+        jdbcTemplate.update("INSERT INTO file_poc (name, value) VALUES ((?), (?))", name, readFileFromResources(file));
```

Comment:
Modify this generic query appropriately; we may want to either make it either more generic a query or have separate methods for different table inserts if there are different scenarios